HelpMyStreetFE/HelpMyStreetFE/Views/Shared/Components/PopulatedDataTable/PopulatedDataTable.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@using HelpMyStreet.Contracts.ReportService;
+@model HelpMyStreetFE.Models.Account.Report.PopulatedDataTableViewModel
```

Comment:
Model here could simply be Chart, to avoid the need for an additional class.
There are probably pros and cons either way...